app/controllers/admin/users_controller.rb

```diff
@@ -40,6 +40,25 @@ def update
       redirect_to "/admin/users/#{params[:id]}"
     end
 
+    def destroy
+      role = params[:role]
+      @user = User.find(params[:user_id])
```

Comment:
**nitpick (non-blocking):** It seems we only need an actual user instance in one location, in other cases `params[:user_id]` would be sufficient. We could avoid the database call unless absolutely necessary by using the id param everywhere and updating the `elsif` to this:

```ruby
elsif User.find(params[:user_id]).remove_role(role)
```

I'm not actually sure saving one potentially unnecessary DB call is worth the decreased readability though, but I think I'd at least move fetching the user to _after_ the `:super_admin` check.